### PR TITLE
Change url validation param in Helm broker

### PR DESCRIPTION
**Pull request by adamwalach**
**Related issue(s)**
https://github.com/kyma-project/kyma/issues/5102

### CODE REVIEW

**Comment by adamwalach** [comment]
After evaluation the config.js looks like this:
```
var urlPrefixes = ["http://"];

urlPrefixes=urlPrefixes.concat(["https://","git::","github.com/","bitbucket.org/"]);

window.clusterConfig = {
  graphqlApiUrl: 'https://console-backend.kyma.local/graphql',
  subscriptionsApiUrl: 'wss://console-backend.kyma.local/graphql',
  DEFAULT_CONFIGURATION_NAME: 'helm-repos-urls',
  HELM_BROKER_REPO_URL_PREFIXES: urlPrefixes
};
```

File: docs/helm-broker/05-01-helm-broker.md [hunk 1]
```diff
@@ -21,4 +21,7 @@ This table lists the configurable parameters, their descriptions, and default va
 | **ctrl.resources.requests.memory** | Defines requests for memory resources. | `32Mi` |
 | **ctrl.tmpDirSizeLimit** | Specifies a size limit on the `tmp` directory in the Helm Pod. This directory is used to store processed addons. Eviction manager monitors the disk space used by the Pod and evicts it when the usage exceeds the limit. Then, the Pod is marked as `Evicted`. The limit is enforced with a time delay, usually about 10s. | `1Gi` |
 | **global.cfgReposUrlName** | Specifies the name of the default ConfigMap which provides the URLs of addons repositories. | `helm-repos-urls` |
-| **global.isDevelopMode** | Defines that each repository URL must be an HTTPS server. If set to `true`, HTTP servers are also acceptable.  | `false` |
+| **global.isDevelopMode** | Defines that each repository URL must be an HTTPS server. If set to `true`, HTTP servers are also acceptable. | `false` |
```
Comment (majakurcius) [inline docs/helm-broker/05-01-helm-broker.md]:
```suggestion
| **global.isDevelopMode** | Defines whether to accept URL prefixes from the **global.urlRepoPrefixes.additionalDevelopMode** list. If set to `true`, the Helm Broker accepts the prefixes from the list. | `false` |
```

**Pull request merged**